### ISSUE TITLE
Crash on Android

### ISSUE THREAD

**Issue by ovr** [getsentry/react-native-sentry#163]
Hey!

```
java.lang.NullPointerException: null
    at io.sentry.RNSentryModule.captureBreadcrumb(RNSentryModule.java:172)
    at java.lang.reflect.Method.invokeNative(Method.java)
    at java.lang.reflect.Method.invoke(Method.java:515)
    at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:363)
    at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:166)
    at com.facebook.react.bridge.queue.NativeRunnable.run(NativeRunnable.java)
    at android.os.Handler.handleCallback(Handler.java:733)
    at android.os.Handler.dispatchMessage(Handler.java:95)
    at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:31)
    at android.os.Looper.loop(Looper.java:146)
    at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:194)
    at java.lang.Thread.run(Thread.java:841)
```

Thank

**Comment by dioxide** [comment]
same problem!

**Comment by ovr** [comment]
refs https://github.com/ovr/ghubber/issues/40

**Comment by HazAT** [comment]
This should be fixed with `0.14.9`
